packages/instant/src/redux/reducer.ts

```diff
@@ -7,13 +7,22 @@ import { AsyncProcessState } from '../types';
 
 import { Action, ActionTypes } from './actions';
 
-export interface State {
+interface BaseState {
     selectedAssetData?: string;
     selectedAssetAmount?: BigNumber;
     selectedAssetBuyState: AsyncProcessState;
     ethUsdPrice?: BigNumber;
     latestBuyQuote?: BuyQuote;
 }
+interface StateWithError extends BaseState {
+    latestError: any;
+    latestErrorDismissed: boolean;
```

Comment:
what if we call this `latestErrorState` and it can be either `Present` or `Hidden` or something. And lets not allow it to be `undefined`.

Reply:
I like this -- feels more clear than undefined or a boolean 💯.  Will update!